inventories/inventory_loop_hosts/playbook_bad.yml

```diff
@@ -1,12 +1,14 @@
-- name: provision hosts in a bad way
+- name: Provision hosts in a bad way
```

Comment:
I think this playbook is supposed to be bad so maybe it shouldn't pass lint?

Reply:
I considered that as well, but was thinking the focus was more around the approach to the problem than about the formatting of the file.  I can change it back if we want.

Reply:
@ericzolf what is your opinion here?

Reply:
I agree with Jeff, let's keep it 100% clean unless we have a very good reason (or the next person going through this exercise will trip over it).